### DISCUSSION TITLE
Is it possible for a route to handle multiple paths?

### DISCUSSION THREAD

**Discussion by Overdash** [actix/actix-web#2580] [Q&A] [answered]
So if I have a route like 

```rust
#[get("/hi")]
fn hello(...) -> ... {...} 
```

how can I get this route to handle incoming traffic for `/hi` and `/hello` (all through `GET`). 
Is this even possible yet? The only work around I can think of is copying a pasting the log in another `fn` that will handle a different path

**Comment by robjtede** [comment, marked as answer]
```rust
async fn handler() -> impl Responder {
    ...
}

App::new()
    .service(web::resource("/route1").route(web::get().to(handler)))
    .service(web::resource("/route2").route(web::get().to(handler)))

// or
App::new()
    .service(web::resource(["/route1", "/route2"]).route(web::get().to(handler)))

```

**Reply by Overdash** [reply]
Ah thanks, I was trying to use it with macros but guess that won't work!
